lib/verifyBearerAuthFactory.js

```diff
@@ -36,10 +35,19 @@ module.exports = function verifyBearerAuthFactory (options) {
         return
       }
       reply.send(errorResponse(noHeaderError))
-      return
+    }
+
+    const header = request.raw.headers.authorization
+    if (!header) {
+      return authorizationHeaderErrorFn('missing authorization header')
     }
 
     const key = header.substring(bearerType.length).trim()
+    const type = header.substring(0, bearerType.length)
+    if (type.toLowerCase() !== bearerType.toLowerCase()) {
```

Comment:
instead of calling `bearerType.toLowerCase()` for each request, could you pre-calculate it when the plugin is being registered and use the const directly?

Reply:
It should be `Bearer` per rfc https://www.rfc-editor.org/rfc/rfc6750#section-2.1